### ISSUE TITLE
High resolution options

### ISSUE THREAD

**Issue by MDMAchine** [MochiDiffusion/MochiDiffusion#117]
### Running Latest Version

- [X] I am running the latest version

### What do you want Mochi Diffusion to do?

While RealESRGAN is a common upscaler to use. I do like to have the option to utilize other models like [BSRGAN](https://github.com/cszn/BSRGAN) for example.

### Why do you think this should be added?

Theres a couple coreml versions [here](https://github.com/john-rocky/CoreML-Models#super-resolution). They all render pretty quickly!

**Comment by calben99** [comment]
Allowing Latent Upscaling would also be super useful!

**Comment by jrittvo** [comment]
Agree, but I'm not sure Apple's frameworks and libraries provide a way to work that directly with the latent space.  Someone else who knows more than I do on this will hopefully give you a more definitive answer.

The BSRGAN module is ready to go.  It is a drop-in replacement for the RealESRGAN that is used now, bu the UI needs a way to let the user select which to use, and that is on the to-do list.

**Comment by jrittvo** [comment]
@stuartjmoore offered to work on UI-centered upgrades.  The heavy coding for this issue is already done, thanks to @vzsg with his build of BSRGAN that can directly substitute for the current RealERSGAN.  (I renamed the BSRGAN file to RealESRGAN and copied it into the project, and it built and ran fine.)

So from here, implementing it means adding a new drop-down in Settings to let the user choose between the two upscalers (and maybe more in the future) and changing the actual code to allow the upscaler to be used to be set from the Settings.  There is the question of whether this belongs here in Mochi or in ml-stable-diffusion, but I don't see the RealESRGAN model in their repo, so I assume it was added as part of assembling Mochi itself.

**Comment by gdbing** [comment]
Upscaling (using RealESRGAN) has been removed from Mochi Diffusion, as of [v6.0](https://github.com/MochiDiffusion/MochiDiffusion/releases/tag/v6.0)

I might backpedal on this if there's significant interest but I'm not interested in upscaling as a feature of Mochi
